ethpm_cli/parser.py

```diff
@@ -141,6 +142,12 @@ def add_package_version_to_parser(
     )
 
 
+def add_uri_to_parser(parser: argparse.ArgumentParser, help_msg: str,) -> None:
```

Comment:
You can remove the trailing comma after `str` here. Trailing commas are typically only useful when there is only one argument per line aka
```python
things =  (
    parser: argparse.ArgumentParser,
    help_msg: str,
)
```